docs/workflow_solutions/using_anaconda.md

```diff
@@ -282,6 +282,94 @@ Now we can be sure that the correct versions of the software will be installed o
     The example above is provided only for illustration purposes. The error has since been fixed, but the example above really happened and is helpful to explain version pinning.
 <!-- markdownlint-enable MD046 -->
 
+#### Good Practice for Finding Software Packages on Anaconda
+
+Finding Anaconda software packages involves searching through the available “Channels” and repositories to locate the specific packages that contain functions that you need for your environment. Channels are locations that store packages, these locations are where Anaconda looks for packages when an installation is to be done. The following are Anaconda Channels that are readily used to house majority of the packages used in scientific research. i. Anaconda ii. Conda-Forge iii. BioConda. There are other Channels also. If you want more information on Anaconda Channels please see their [docs](https://docs.anaconda.com/).
+
+##### Step-by-Step Guide to Finding Anaconda Software Packages
+
```

Comment:
Maybe have a summary paragraph here that describes the main steps. (1) Find key packages. (2) Ensure they are up-to-date. (3) Figure out the best way to install them. (4) Compose an environment file.